spec/rubocop/cop/style/braces_around_hash_parameters_spec.rb

```diff
@@ -261,6 +261,14 @@
         expect(corrected).to eq('get :i, x: 1')
       end
     end
+
+    context 'in a method call with multi line arguments without parentheses' do
+      it 'removes hash braces' do
+        src = "render 'foo', {\n  foo: bar\n}"
+        corrected = autocorrect_source(src)
+        expect(corrected).to eq("render 'foo', \n  foo: bar\n")
```

Comment:
I assume that this wasn't working before the change. It seems like this is what should be in the CHANGELOG. We normally only add CHANGELOG references for changes that could impact users. The average user doesn't know what `RangeHelp`  or `range_by_whole_lines` is.

Reply:
Sounds good to me, do you have a preferred wording for the changelog?

Reply:
That sounds fine to me. Thanks for adding some context. Do you know what in particular about this line of code caused it go past the end of the file? Maybe something about the spacing or new lines?

Reply:
the example broken source is on line 267
```
src = "render 'foo', {\n  foo: bar\n}"
```
the problem is that this cop wants to remove the braces around hash argument, and to do that it will try to select the entire line where `}` is located including a possible trailing newline, to do so `range_by_whole_lines` will add 1 character to the end of the line here:
https://github.com/bbatsov/rubocop/blob/a66faa88ce004f8717aff136230d196e8be3ee2c/lib/rubocop/cop/mixin/range_help.rb#L68
and this makes the range go past the end of the buffer because `}` in this example is the last character in the buffer.

Reply:
It sounds like the issue comes from a closing curly bracket, `}` living on a separate line than the contents of the hash. If that is the case, it would be useful to add to the CHANGELOG.

Reply:
Added some clarification to changelog 👍